validators/src/unit-test/java/uk/gov/moj/sdt/validators/BulkSubmissionValidatorTest.java

```diff
@@ -157,17 +167,12 @@ public class BulkSubmissionValidatorTest extends AbstractValidatorUnitTest {
     /**
      * Setup of the Validator and Domain class instance.
      */
+    @BeforeEach
     public void setUpLocalTests() {
 
         // subject of test
         validator = new BulkSubmissionValidator();
-        validator.setConcurrencyMap(new HashMap<String, IInFlightMessage>());
-
-        // mock BulkCustomer object
-        mockIBulkCustomerDao = EasyMock.createMock(IBulkCustomerDao.class);
-
-        globalParameterCache = EasyMock.createMock(ICacheable.class);
-
+        validator.setConcurrencyMap(new HashMap<>());
```

Comment:
Should this method contain MockitoAnnotations.openMocks(this); to initialise the mocked objects?

Reply:
It appears that annotating the class with @ExtendWith(MockitoExtension.class) automatically sets up the mocks, so this isn't needed.